code/ui/ui_playermodel.c

```diff
@@ -1708,7 +1713,8 @@ void PlayerModel_DataInit(void){
     qsort((void *)s_playermodel.genderList, s_playermodel.numGenders + 1, sizeof(filterData_t), FilterList_Compare);
 
     //Populate the spin control pointers
-    for (i = 0; i < 128; i++) {
+    int end = min(MAX_RACES, MAX_GENDERS);
```

Comment:
min is not defined. Added it to top of file. Should be in some sort of common file but this fixes build errors for now.